From be8e785db92c1cb448943b908900b324c93b5b08 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Uwe=20Kleine-K=C3=B6nig?= <ukleinek@kernel.org>
Date: Wed, 29 Jul 2026 14:16:40 +0200
Subject: [PATCH 14518/15333] Drop Michael Turquette's clk maintainer entry
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Mike wasn't publically active for the clk subsystem since years. Stop
pretending that there is a maintainer team caring for clk.

Signed-off-by: Uwe Kleine-König <ukleinek@kernel.org>
Acked-by: Michael Turquette <mturquette@baylibre.com>
Signed-off-by: Stephen Boyd <sboyd@kernel.org>
---
 CREDITS     | 4 ++++
 MAINTAINERS | 1 -
 2 files changed, 4 insertions(+), 1 deletion(-)

diff --git a/CREDITS b/CREDITS
index 84793a967a0b..c8a67ab8efe5 100644
--- a/CREDITS
+++ b/CREDITS
@@ -4085,6 +4085,10 @@ S: 4226 Landgreen Street
 S: Rockville, Maryland 20853
 S: USA
 
+N: Michael Turquette
+E: mturquette@baylibre.com
+D: common clk maintenance
+
 N: Stephen Tweedie
 E: sct@redhat.com
 P: 1024/E7A417AD E2 FE A4 20 34 EC ED FC 7D 7E 67 8D E0 31 D1 69
diff --git a/MAINTAINERS b/MAINTAINERS
index d15f20dbc92d..6654ed8a17ad 100644
--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -6447,7 +6447,6 @@ F:	include/linux/comedi/
 F:	include/uapi/linux/comedi.h
 
 COMMON CLK FRAMEWORK
-M:	Michael Turquette <mturquette@baylibre.com>
 M:	Stephen Boyd <sboyd@kernel.org>
 R:	Brian Masney <bmasney@redhat.com>
 L:	linux-clk@vger.kernel.org
-- 
2.55.0

